spec/researches/keywordCountSpec.js

```diff
@@ -2,41 +2,113 @@
+		expect( keywordCount( mockPaper, mockResearcherGermanDiacritics ).count ).toBe( 1 );
+	} );
+
+	it( "counts a string with multiple keyword morphological forms", function() {
+		const mockPaper = new Paper( "A string of text with a keyword and multiple keywords in it." );
+		expect( keywordCount( mockPaper, mockResearcher ).count ).toBe( 2 );
+	} );
+
+	it( "counts a string with multiple keywords", function() {
+		const mockPaper = new Paper( "A string with strawberries and a keyword." );
+		expect( keywordCount( mockPaper, mockResearcherMultipleKeywords ).count ).toBe( 1 );
+	} );
+
+	it( "counts a string with a keyword with a '-' in it", function() {
+		const mockPaper = new Paper( "A string with a key-word." );
+		expect( keywordCount( mockPaper, mockResearcherMinus ).count ).toBe( 1 );
+	} );
+
+	it( "does not count a string with 'key-word' in it, if the keyword is 'key word'", function() {
+		const mockPaper = new Paper( "A string with a key-word." );
+		expect( keywordCount( mockPaper, mockResearcherKeyWord ).count ).toBe( 1 );
+	} );
+
+	it( "counts a string with a keyword with a '_' in it", function() {
+		const mockPaper = new Paper( "A string with a key_word." );
+		expect( keywordCount( mockPaper, mockResearcherUnderscore ).count ).toBe( 1 );
+	} );
+
+	it( "counts a string with with 'kapaklı' as a keyword in it", function() {
+		const mockPaper = new Paper( "A string with kapaklı." );
+		expect( keywordCount( mockPaper, mockResearcherKaplaki ).count ).toBe( 1 );
+	} );
+
+	it( "counts a string with with '&' in the string and the keyword", function() {
+		const mockPaper = new Paper( "A string with key&word" );
+		expect( keywordCount( mockPaper, mockResearcherAmpersand ).count ).toBe( 1 );
+	} );
+
+	it( "does not count images as keywords.", function() {
+		const mockPaper = new Paper( "<img src='http://image.com/image.png'>" );
+		expect( keywordCount( mockPaper, mockResearcherAmpersand ).count ).toBe( 0 );
+	} );
+
+	it( "keyword counting is blind to CApiTal LeTteRs.", function() {
+		const mockPaper = new Paper( "A string with KeY worD." );
+		expect( keywordCount( mockPaper, mockResearcherKeyWord ).count ).toBe( 1 );
+	} );
+
+	it( "keyword counting is blind to types of apostrophe.", function() {
+		const mockPaper = new Paper( "a string with quotes to match the key'word, even if the quotes differ" );
+		expect( keywordCount( mockPaper, mockResearcherApostrophe ).count ).toBe( 1 );
+	} );
+
+	it( "counts can count dollar sign as in '$keyword'.", function() {
+		const mockPaper = new Paper( "a string with a $keyword" );
+		expect( keywordCount( mockPaper, mockResearcherDollarSign ).count ).toBe( 1 );
+	} );
+
+	it( "counts a string of text with a keyword in it once, even if the text contains 'key-word' as well.", function() {
+		const mockPaper = new Paper( "Lorem ipsum dolor sit amet, key word consectetur key-word adipiscing elit " );
+		expect( keywordCount( mockPaper, mockResearcherKeyWord ).count ).toBe( 2 );
```

Comment:
Same as above: `key word` is counted twice, once for `key word` and once for `key-word`. The description says the count should be 1.